content/v1.14/cli/command-reference.md

```diff
@@ -0,0 +1,416 @@
+---
+weight: 50
+title: Command Reference
+description: "Command reference for the Crossplane CLI"
+---
+
+<!-- vale Google.Headings = NO -->
+The `crossplane` CLI provides utilities to make using Crossplane easier. 
+
+Read the [Crossplane CLI overview]({{<ref "../cli">}}) page for information on 
+installing `crossplane`.
+
+## Global flags
+The following flags are available for all commands.
+
+{{< table "table table-sm table-striped">}}
+| Short flag | Long flag   | Description                  |
+|------------|-------------|------------------------------|
+| `-h`       | `--help`    | Show context sensitive help. |
+| `-v`       | `--version` | Print version and exit.      |
+|            | `--verbose` | Print verbose output.        |
+{{< /table >}}
+
+## xpkg
+
+The `crossplane xpkg` commands create, install and update Crossplane
+[packages]({{<ref "../concepts/packages">}}) as well as enable authentication
+and publishing of Crossplane packages to a Crossplane package registry. 
+
+### xpkg build
+
+Using `crossplane xpkg build` provides automation and simplification to build 
+Crossplane packages.  
+
+The Crossplane CLI creates a 
+single YAML file from a directory of YAML files and combines them as an 
+[OCI container image](https://opencontainers.org/).  
```

Comment:
Does it matter to the user/reader at this level that it creates a single YAML file? That seems to me like an internal implementation detail. It's covered by linking to the spec for those who really want to know how it works. Saying it combines them into a package as you do below feels like the right level of detail to me.

Reply:
The reason I call this out is that if you've read the spec it talks about a single yaml file. I'm calling out "hey, this makes things better because you don't have to build the single massive yaml, we'll do it for you"

Reply:
I don't think we should be assuming users of `crossplane xpkg` build are familiar with the spec. That feels a bit like assuming `docker build` users have read the OCI specification and are thinking about what it does through that lens.

Reply:
updated.